resources/user.rb

```diff
@@ -0,0 +1,106 @@
+property :username, String, name_attribute: true, desired_state: false, identity: true
```

Comment:
Please use `name_property` instead of `name_attribute`.
In addition, `name_property` does not need to be marked as `desired_state` nor `identity`.

Reply:
done